check-diff.sh

```diff
@@ -776,6 +792,36 @@ function lint_js_files {
 	fi
 }
 
+
+function lint_css_files {
+	if [ ! -s "$TEMP_DIRECTORY/paths-scope-css" ]; then
+		return
+	fi
+
+	set -e
+
+	# Run ESLint.
+	if [ -n "$STYLELINT_CONFIG" ] && [ -e "$STYLELINT_CONFIG" ] && [ -e "$(npm bin)/stylelint" ] && check_should_execute 'stylelint'; then
+		(
+			echo "## stylelint"
+			cd "$LINTING_DIRECTORY"
+			# TODO: --format=compact is not right.
+			if ! cat "$TEMP_DIRECTORY/paths-scope-css" | remove_diff_range | xargs "$(npm bin)/stylelint" --format=compact --config="$STYLELINT_CONFIG" > "$TEMP_DIRECTORY/stylelint-report"; then
```

Comment:
It seems that it actually needs to be `--custom-formatter node_modules/stylelint-formatter-compact`